receiver/aerospikereceiver/README.md

```diff
@@ -18,18 +18,25 @@ Aerospike versions 4.9, 5.x, and 6.x are supported.
 
 Configuration parameters:
 
-- `endpoint` (default localhost:3000)
-- `collect_cluster_metrics` (default false): Whether discovered peer nodes should be collected
+- `endpoint` Map of Aerospike host name, port, and tls name to connect to.
+  - `name` Host name or address.
+  - `port` Host port number.
+  - `tlsname` Host tls name. Used by the client during TLS connections. See [Aerospike authentication](https://docs.aerospike.com/server/guide/security/tls#standard-authentication) for mor details.
+- `collect_cluster_metrics` (default false): Whether discovered peer nodes should be collected.
 - `collection_interval` (default = 60s): This receiver collects metrics on an interval. Valid time units are ns, us (or µs), ms, s, m, h.
 - `username` (Enterprise Edition only.)
 - `password` (Enterprise Edition only.)
+- `tls` (default empty/no tls) tls configuration for connection to Aerospike nodes. More information at [OpenTelemetry's tls config page](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md).
 
 ### Example Configuration
 
 ```yaml
 receivers:
     aerospike:
-        endpoint: localhost:9000
+        endpoint:
+            name: "localhost"
+            port: 3000
+            tlsname: ""
```

Comment:
There's a pretty well established pattern across collector components of having `endpoint` be a string. If at all reasonable, I think it would be better to keep following that standard.

Does `tlsname` really need to be within/under `endpoint`? I get that they are associated with each other, but could say the same of almost any setting in the receiver configuration. Could this just be an additive change at the same level as endpoint?
```yaml
aerospike:
  endpoint: host:port
  tlsname: foo
```

Reply:
Got it. I went with this because it maps to our Go client's host structure exactly so no extra parsing is needed. I'm willing to go with the additive change you suggested though. I will make the change soon.